src/utils/shared.ts

```diff
@@ -18,12 +18,14 @@ export const toHtmlSafeId = (
 };
 
 export const validateSearchField = (value?: string) => {
-  return value ? /^([a-zA-Z0-9-_%]*[a-zA-Z0-9-_%])?$/.test(value.trim()) : true;
+  return value
+    ? /^([a-zA-Z0-9-_%_]*[a-zA-Z0-9-_%_])?$/.test(value.trim())
+    : true;
 };
 
 export const validateConnectorSearchField = (value?: string) => {
```

Comment:
@mmuzikar `validateConnectorSearchField` function is for the Connector Type search and the other `validateSearchField` is for the rest of the search fields.